packages/dd-trace/src/ci-visibility/log-submission/log-submission-plugin.js

```diff
@@ -0,0 +1,49 @@
+const Plugin = require('../../plugins/plugin')
+const log = require('../../log')
+
+function getWinstonLogSubmissionParameters (config) {
+  const { site, service } = config
+
+  const defaultParameters = {
+    host: `http-intake.logs.${site}`,
+    path: `/api/v2/logs?dd-api-key=${process.env.DD_API_KEY}&ddsource=winston&service=${service}`,
```

Comment:
I think it is safer to include the API key as a request header (`dd-api-key`), rather than as part of the query string.

Reply:
Although taking a closer look at this, I am not sure if this is what `winston` allows.

Reply:
safer because there's less likelihood of the API being logged somewhere, right? 

It is possible to set headers, yeah. I was following the instructions shown by APM for the APM-logs integration, which are more tested: https://docs.datadoghq.com/logs/log_collection/nodejs/?tab=winston30#agentless-logging. In this case I think it's worth doing the change though

Reply:
> safer because there's less likelihood of the API key being logged somewhere
yep, this is what I had in mind

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/4762/commits/0d70afb92e060a5c11b08a5a1709d8f0e003e3d8